src/main/java/org/folio/spring/SpringContextUtil.java

```diff
@@ -40,22 +42,28 @@ public static void init(Vertx vertx, Context context) {
    */
   public static void init(Vertx vertx, Context context, Class<?> defaultConfiguration) {
     String configClassName = context.config().getString(SPRING_CONFIGURATION);
+    log.debug("init:: Attempts to create spring base context");
+
     if (configClassName != null) {
+      log.info("init:: configClassName != null");
       try {
         Class<?> springConfigClass = Class.forName(configClassName);
         AnnotationConfigApplicationContext springContext = createBaseContext(vertx, context);
         springContext.register(springConfigClass);
         springContext.refresh();
         context.put(SPRING_CONTEXT_KEY, springContext);
       } catch (ClassNotFoundException e) {
-        throw new IllegalStateException("Failed to load configuration class " + configClassName);
+        log.warn("Failed to load configuration class, msg: {}" + e.getMessage());
+        throw new IllegalStateException();
       }
     } else if (defaultConfiguration != null) {
+      log.info("init:: defaultConfiguration != null");
```

Comment:
maybe something like this? 
```suggestion
      log.info("init:: Attempts to create spring context by defaultConfiguration");
```

Reply:
Ok, than line 66 should be like `init:: configClassName & defaultConfiguration == null`